Sofa/GUI/Qt/src/sofa/gui/qt/RealGUI.cpp

```diff
@@ -226,8 +227,23 @@ class RealGUIFileListener : public sofa::helper::system::FileEventListener
 
 //======================= STATIC METHODS ========================= {
 
+void RealGUI::setupSurfaceFormat()
+{
+    if(!SOFA_GUI_QT_ENABLE_VSYNC)
+    {
+        static std::once_flag flag;
+        std::call_once(flag, []
+        {
+            QSurfaceFormat format;
+            format.setSwapInterval(0);
+            QSurfaceFormat::setDefaultFormat(format);
```

Comment:
Should we not set here the  compatibility profile and so on ?
(and the MSAA sampling here but we dont have the arg here unfortunately)

Reply:
Compatibility profile is set only in QtViewer, right?
QtViewer setups a new `QSurfaceFormat` based on the default format, which was modified previously to support Vsync (or not). So, I guess that for QtViewer the compatibility profile is set properly. What do you think?
Do you think all this setup must go in the default QSurfaceFormat?

Reply:
Humm I was thinking just to uniformize/factorize/gather in one location the surfaceFormat initialization (both qt viewers would share the same surfaceformat once for all)

Reply:
Done. Let me know if it matches your thoughts